includes/Admin/Editor.php

```diff
@@ -279,15 +284,17 @@ public function admin_enqueue_scripts( $hook ) {
 			true
 		);
 
-		$script_dependencies = [ Tracking::SCRIPT_HANDLE, 'postbox', self::AMP_VALIDATOR_SCRIPT_HANDLE ];
-
-		$this->assets->enqueue_script_asset( self::SCRIPT_HANDLE, $script_dependencies, false );
+		wp_enqueue_script( self::SCRIPT_HANDLE );
 		$this->assets->enqueue_style_asset( self::SCRIPT_HANDLE, [ $this->google_fonts::SCRIPT_HANDLE ] );
 
 		wp_localize_script(
 			self::SCRIPT_HANDLE,
-			'webStoriesEditorSettings',
-			$this->get_editor_settings()
+			'webStories',
+			[
+				'publicPath' => $this->assets->get_base_url( 'assets/js/' ), // Required before the editor script is enqueued.
+				'localeData' => $this->assets->get_translations( self::SCRIPT_HANDLE ), // Required for i18n setLocaleData.
```

Comment:
As an alternative we could think about exposing `setLocaleData` under `window.webStories.setLocaleData`, and then make these `setLocaleData` calls inline. Then we wouldn't to pass this huge list around.

Not sure that's really an advantage though 🤔

Reply:
But shouldn't `setLocaleData(localeData);` be called before the script is loaded. 
comment 👇 

https://github.com/google/web-stories-wp/blob/d7c7f712510516c13277d128fd907080dab7bfdd/packages/wp-story-editor/src/index.js#L26-L28

and `setLocalData` is included in the package, meaning it not be available before the script is loaded.

https://github.com/google/web-stories-wp/blob/d7c7f712510516c13277d128fd907080dab7bfdd/packages/wp-story-editor/src/setLocaleData.js#L20

So how would can we add an inline script?

Reply:
Ah, you're right 🤦  never mind then